HW_1/task_4.py

```diff
@@ -0,0 +1,17 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов, а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+# *Пример:*
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+#     60 -> 10  40  10
+print("Введите кол-во журавликов:")
+s = int(input())
+if s % 6 == 0 and s > 0: # следует из условия задачи
+    Pet = s // 6
+    Ser = Pet
```

Comment:
```suggestion
    Pet = Ser = s // 6
```
А еще в Python не принято начинать имена переменных функций и так далее. Исключение, конечно же, составляют имена классов с символа верхнем регистре.